src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+type Props = {
+  userName: string;
+  userEmail: string;
+};
+
+export const UserInfo: React.FC<Props> = ({ userName, userEmail }) => {
```

Comment:
```suggestion
export const UserInfo: React.FC<Props> = ({ name, email }) => {
```
Make sure to name props properly to pass tests